src/originalBudgetTrackingApp/commonTypes.ts

```diff
@@ -1,7 +1,15 @@
-import { Transaction } from '@/originalBudgetTrackingApp/bankScraper';
+import { Config } from './configManager/configManager';
+import { Transaction } from './bankScraper';
 
 export interface EnrichedTransaction extends Transaction {
   accountNumber: string;
   category?: string;
   hash: string;
 }
+
+export interface OutputVendor {
+  name: string;
+  init?: (config: Config) => Promise<void>;
+  exportTransactions: (transactionsToCreate: EnrichedTransaction[], startDate: Date, config: Config) => Promise<any>;
+  isActive: (config: Config) => boolean;
```

Comment:
This is strange. I also wondered about that, and I realized that maybe we don't need to use `isActive` at all.
Think about that, we have a method that takes `config` and does scraping (and we have completely different module to save the `config`), so if you don't want to export to somewhere, you just need to send the config without it.

Reply:
Now I'm thinking again. Maybe in the theory, we don't need that, but in real life, we will prefer to set `enabled: false` instead of removing it from `config` and saving it somewhere, just to test something.

So it should be a field in `config`, not method in the `outputVendor`.

Reply:
You mean the export transactions would check the relevant flag in the config and skip if it is not enabled instead of having the check if active or not happen before calling exportTransactions?

Thats fine, the only drawback is that every output vendor needs to remember to check this flag

Reply:
Another option is to have the config's of the output vendors extend an interface that will have the active boolean field, then the calling code can get for each output vendor it's config and check the active boolean.

Reply:
Pushed the solution that takes it from the config but still in a generic way